packages/calcite-components/src/components/label/label.e2e.ts

```diff
@@ -145,6 +146,44 @@ describe("calcite-label", () => {
     expect(element).toEqualAttribute("layout", "inline-space-between");
   });
 
+  it("should focus on the slotted form element when a label is clicked", async () => {
```

Comment:
I like how this provides context for the selection test, but it is already covered by the `labelable` helper. 

Maybe it's OK to merge with the selection test where it does the following

1. text is selected + click > no focus shift
2. text is unselected + click > focus shifts

?

Reply:
I can remove this test since its testing duplicate functionality.